### ISSUE TITLE
Best configuration for finetuning

### ISSUE THREAD

**Issue by Jogima-cyber** [leondgarse/Keras_efficientnet_v2_test#4]
Do you have an idea for the best configuration for finetuning ? Which optimizer, which LR reduction strategy, need for dropout, regularization, need for weird learning technics?

**Comment by Jogima-cyber** [comment]
Btw for my current configuration I'm using this LR reducer with adam optimizer, LR set to 0.001, and regular tf training : 
```python
lr_reducer = tf.keras.callbacks.ReduceLROnPlateau(
        monitor="val_classification_loss", patience=3, min_lr=1e-6, mode='min')
```

**Comment by leondgarse** [comment]
- I didn't train much on them. Saying the optimizer, many articles and my own practices result that `SGD + momentum` works better than `adam` on finetuning. You can also check `SGDW` / `AdamW` from [tensorflow-addons](https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers). It also seems better using `tfa.optimizers.Lookahead` when finetuning.
- The lr scheduler, recently I'm mainly using `tf.keras.experimental.CosineDecayRestarts` for `SGD`, also starts from `initial_learning_rate=1e-3` for finetuning.
- Other weird technics like `StochasticDepth` also comes from `tensorflow-addons` may also help:
  ```py
  model = efficientnet_v2.EfficientNetV2L(input_shape=(None, None, 3), survivals=(1, 0.8), dropout=1e-6, classes=1000)
  ```
- `Dropout` or other `regularization`, I hadn't tested on them...

**Comment by Jogima-cyber** [comment]
Thank you very much for your insights!
